### PR TITLE
Improve testing time

### PR DESCRIPTION
**Pull request by SkafteNicki**
## Before submitting

- [x] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure to update the docs?
- [ ] Did you write any new necessary tests?

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by SkafteNicki** [comment]
Not ready for review, just need all tests to run to see effect of changes.

File: tests/helpers/testers.py [hunk 1]
```diff
@@ -33,7 +33,7 @@
     pass
 
 NUM_PROCESSES = 2
-NUM_BATCHES = 10
+NUM_BATCHES = 8
```
Comment (SkafteNicki) [inline tests/helpers/testers.py:36]:
```suggestion
NUM_BATCHES = 8  # Need to be divisible with the number of processes
```

File: tests/helpers/testers.py [hunk 2]
```diff
@@ -33,7 +33,7 @@
     pass
 
 NUM_PROCESSES = 2
-NUM_BATCHES = 10
+NUM_BATCHES = 8  # Need to be divisible with the number of processes
```
Comment (Borda) [inline tests/helpers/testers.py:36]: shall we co even with a smaller number?
```suggestion
NUM_BATCHES = 4  # Need to be divisible with the number of processes
```

Comment (SkafteNicki) [reply tests/helpers/testers.py:36]: lets try

File: requirements/image.txt [hunk 3]
```diff
@@ -2,3 +2,4 @@ scipy
 torchvision  # this is needed to internally set TV version according installed PT
 torch-fidelity
 lpips
+PIL==8.4.0
```
Comment (SkafteNicki) [inline requirements/image.txt:5]:
```suggestion
Pillow==8.4.0
```

File: .github/workflows/ci_test-full.yml [hunk 4]
```diff
@@ -56,6 +56,7 @@ jobs:
     env:
       PYTEST_ARTEFACT: test-results-${{ matrix.os }}-py${{ matrix.python-version }}-${{ matrix.requires }}.xml
       PYTORCH_URL: https://download.pytorch.org/whl/cpu/torch_stable.html
+      TRANSFORMERS_CACHE: .cache/huggingface/
```
Comment (Borda) [inline .github/workflows/ci_test-full.yml:59]:
```suggestion
```
this is not needed

File: requirements/image.txt [hunk 5]
```diff
@@ -2,3 +2,4 @@ scipy
 torchvision  # this is needed to internally set TV version according installed PT
 torch-fidelity
 lpips
+Pillow==8.4.0
```
Comment (Borda) [inline requirements/image.txt:5]: only one version or could it be some tide range?

Comment (SkafteNicki) [reply requirements/image.txt:5]: This is the latest release need for it to work. Do older packages may also

File: .github/workflows/ci_test-full.yml (hunk 4 above)
Comment (SkafteNicki) [reply .github/workflows/ci_test-full.yml:59]: It was apparently needed to get some of the Bert score tests unstuck

Comment (Borda) [reply .github/workflows/ci_test-full.yml:59]: strange as this has to fail on Win :)

File: requirements/image.txt (hunk 5 above)
Comment (Borda) [reply requirements/image.txt:5]: how much older?

**Review by Borda — APPROVED** [review]

**Review by awaelchli — APPROVED** [review]

**Review by tchaton — APPROVED** [review]

**Pull request merged**